test/stats.spec.ts

```diff
@@ -7,6 +7,12 @@ const expectedThrow = {
   back: 2,
   down: 1,
 };
+const expectedTech = {
+  backward: 2,
+  forward: 1,
+  neutral: 8,
+  fail: 4,
+};
```

Comment:
Can you move this inside the tech test function itself like in line 145?

Reply:
yep